.github/workflows/tests_with_dev_pl.yml

```diff
@@ -0,0 +1,95 @@
+name: Testing
```

Comment:
Warning: lots of repetition with `tests.yml`, see comment below for differences.